app/views/api/notes/_note.xml.builder

```diff
@@ -14,6 +14,10 @@ xml.note("lon" => note.lon, "lat" => note.lat) do
 
   xml.date_closed note.closed_at if note.closed?
 
+  note.tags.each do |k, v|
```

Comment:
This code is duplicated multiple times, consider creating a shared helper method to reuse instead.